### PR TITLE
Update arm image tag

### PR DESCRIPTION
**Pull request by dramich**
Update the arm image tag
Removed the push from the Build step so push can push
Removed the if check since the whole job only runs on tag

### CODE REVIEW

**Review by paynejacob — APPROVED** [review]

**Review by paynejacob — APPROVED** [review]

**Comment by dramich** [comment]
This PR should hold until the k3s stuff is moved out and we can verify the build succeeds

**Comment by mudler** [comment]
Can you try current images from master? Images tagged with `latest` should be fine by now

File: .github/workflows/release-arm.yaml [hunk 1]
```diff
@@ -53,10 +53,9 @@ jobs:
           ./earthly.sh +all-arm --IMAGE_NAME=c3os-$FLAVOR-$TAG.img --IMAGE=quay.io/c3os/core-$FLAVOR:$TAG --MODEL=$MODEL --FLAVOR=$FLAVOR
           docker push quay.io/c3os/core-$FLAVOR:$TAG
       - name: Push  🔧
-        if: startsWith(github.ref, 'refs/tags/')
         env:
           FLAVOR: ${{ matrix.flavor }}
-          IMAGE: quay.io/c3os/c3os:${{ matrix.flavor }}-latest
+          IMAGE: quay.io/c3os/c3os-${{ matrix.flavor }}:latest
```
Comment (mudler) [inline .github/workflows/release-arm.yaml:58]: I think this is not needed anymore there is a separate workflow pushing arm images now https://github.com/c3os-io/c3os/blob/b81e17ee026628a6cae70a6b0b1ff808145b99bf/.github/workflows/image-arm.yaml#L47

**Review by mudler — CHANGES_REQUESTED** [review]

**Comment by dramich** [comment]
Closed in favor of https://github.com/c3os-io/c3os/pull/86

**Pull request closed without merging**